packages/core/src/components/TextField/Styled/HelperAndErrorTextTooltipWrapper.styled.tsx

```diff
@@ -1,5 +1,7 @@
 import styled from 'styled-components';
 
-export const HelperAndErrorTextTooltipWrapper = styled.div<{ displayCharacterCount: boolean; size: 'S' | 'M' }>`
-    ${({ displayCharacterCount, size }) => (!displayCharacterCount || size !== 'S') && 'margin-left: 1rem;'}
+export const HelperAndErrorTextTooltipWrapper = styled.div<{ displayCharacterCount: boolean; size: 'S' | 'M'; isSuffixPresent: boolean}>`
+    margin-top: 0.5rem;
+    ${({ isSuffixPresent }) => !isSuffixPresent && 'margin-right: 1rem;'}
```

Comment:
We should always write like this -> `margin-right: ${({ isSuffixPresent }) => !isSuffixPresent &&  1rem}`, so that stylelint can catch error if there is any.